README.md

```diff
@@ -11,10 +11,6 @@ Test Pilot is an opt-in platform that allows us to perform controlled tests of n
 
 Test Pilot is not intended to replace trains for most features, nor is it a test bed for concepts we do not believe have a strong chance of shipping in general release. Rather, it is reserved for features that require user feedback, testing, and tuning before they ship with the browser.
 
-## Repositories
-
-* **testpilot** - Test Pilot server and front-end
-* [testpilot-splash](https://github.com/mozilla/testpilot-splash/) - Teaser site for collecting emails
```

Comment:
This link 404s. I don't think changing the link to https://github.com/mozilla/idea-town-splash/ adds any value. :scissors: